lib/safira_web/controllers/cv_controller.ex

```diff
@@ -0,0 +1,56 @@
+defmodule SafiraWeb.CVController do
+  use SafiraWeb, :controller
+
+  alias Safira.Accounts
+
+  alias Safira.CV
+
+  action_fallback SafiraWeb.FallbackController
+
+  def company_cvs(conn, %{"id" => company_id}) do
+    curr_user = Accounts.get_user(conn)
+
+    curr_company_id =
+      curr_user
+      |> Map.get(:company)
+      |> case do
+        nil -> nil
+        company -> Map.get(company, :id)
+      end
+
+    company = Accounts.get_company!(company_id)
```

Comment:
This can fail if the provided `id` is wrong and will throw an error.
From what I could see, it's how it is done anywhere else in the app, and there isn't a version of `get_company` that doesn't raise an exception.